src/openrct2-ui/windows/Ride.cpp

```diff
@@ -1236,6 +1236,19 @@ struct RideTypeLabel
 static sint32 RideDropdownDataLanguage = LANGUAGE_UNDEFINED;
 static std::vector<RideTypeLabel> RideDropdownData;
 
+// Used for sorting the vehicle type dropdown.
+struct VehicleTypeLabel
```

Comment:
This looks the same as `RideTypeLabel`. Can you rename it to just TypeLabel and use them for both instances?

Reply:
The first property is different. I considered templating it, but I didn't think it was worth writing generics whenever it's used.

Reply:
Alright! I was mostly just wondering if you considered it.